style.css

```diff
@@ -37,6 +42,58 @@ section {
   margin-bottom: 80px;
 }
 
+a:link {
+  color: #396df2;
+}
+
+a:hover {
+  color: coral;
+  text-decoration-line: line-through;
+}
+
+a:visited {
+  color: #7f8cff;
+  text-decoration-line: line-through;
+}
+
+.submit {
+  padding: 16px;
+  border-radius: 16px;
+  background: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  display: inline-block;
+  text-decoration: none;
+  position: relative;
+  border: 1px solid #6070ff;
+  order: 3;
+  transition-duration: 0.4s;
+}
+
+.submit:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+}
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly make sure to implement the `hover` effect on your buttons to create a good user experience and to also make your design align with that of the `Figma` design template. I noticed you have added the `hover` effect but it's not working because you have specified a `[background](background: #fff)` on your `button`  at line `87`.
 
 - [ ] To fix this issue, Kindly consider removing the background you have specified on line `87``(.submit:enabled)` and add a color of `#fff` when on `hover`. See below for more details: 
<img width="697" alt="Screenshot 2023-06-07 at 11 22 13" src="https://github.com/ethred/Portfolio/assets/107192076/3e3bf483-b9f9-4dbb-806d-e8c284f438f0">